src/components/OrdersWidget/index.tsx

```diff
@@ -222,36 +234,14 @@ const OrdersWidget: React.FC = () => {
                 <div className="title">Expires</div>
               </div>
 
-              <OrderRow
-                id="1"
-                sellToken={DAI}
-                buyToken={TUSD}
-                price="1.05"
-                unfilledAmount="1,500"
-                availableAmount="1,000"
-                overBalance
-                expiresOn="In 3 min"
-              />
-              <OrderRow id="543" sellToken={TUSD} buyToken={USDC} price="1.03" availableAmount="1,000" unlimited />
-              <OrderRow
-                id="1287"
-                sellToken={USDC}
-                buyToken={DAI}
-                price="1.50"
-                unfilledAmount="876.849"
-                availableAmount="876.849"
-                expiresOn="In 1 days"
-                pending
-              />
-              <OrderRow
-                id="1257"
-                sellToken={PAX}
-                buyToken={DAI}
-                price="1.10"
-                unfilledAmount="5,876.8429"
-                availableAmount="500"
-                expiresOn="In 2 days"
-              />
+              {orders.map(order => (
+                <OrderRow
+                  key={order.id}
```

Comment:
Is this the only place you use `order.id`? Then really no need for that, just use array index.

Reply:
As I said in another comment, the main reason for having the id is for sorting, and later possibly pagination. Let's say `orders` is sorted by expiration date, how else could we keep track of the initial ids?

Reply:
in the real implementation, what will be the id? I think to uniquely refer to an order you need something like:
`key={order.owner + '_' + order.orderId}`

Reply:
Yes, the order the ids are returned is the orderId itself, if I understood the contract correctly: 
https://github.com/gnosis/dex-contracts/blob/master/contracts/BatchExchange.sol#L375 and
https://github.com/gnosis/dex-contracts/blob/master/contracts/BatchExchange.sol#L212

You can only cancel your own orders, so the ids always refer to yourself, so I don't think we need to prefix it with order.owner.

Reply:
Mm, yes, but I would identify the orders using the unique id. Otherwise, you'll find an issue if you reuse your rows for displaying for example the orders of a batch

Reply:
What would be the use case of displaying orders of a batch? Is this a requirement for the short term?
Otherwise, I don't yet see the need to complicate the orderId.

Unless you mean to preemptively adapt to [getEncodedOrders](https://github.com/gnosis/dex-contracts/blob/master/contracts/BatchExchange.sol#L386) (not implemented on frontend API yet), which returns the orders sorted by user.

Still, that's a different context and there's no problem the id being different in these two situations. What I would change for this is to make the `id` property a `string` instead of `number`.

Does that work for you @anxolin ?